shared/src/extensions/ExtensionStatus.tsx

```diff
@@ -100,13 +110,30 @@ export class ExtensionStatus extends React.PureComponent<Props, State> {
                         title="Toggle extension trace logging to devtools console"
                     />
                 </div>
+                <div className="card-body border-top">
+                    <div className="form-group">
+                        <label htmlFor="extension-status__unpackedurl">Load unpacked extension</label>
+                        <input
+                            type="text"
+                            id="extension-status__unpackedurl"
+                            className="form-control"
+                            onChange={this.onUnpackedURLChange}
+                            placeholder="URL"
```

Comment:
```suggestion
                            placeholder="URL to extension's package.json"
```

And then modify the fetchUnpackedExtension code above slightly to treat the user's input URL as the package.json URL instead of appending as in `${baseUrl}/package.json`.

This resolves the user's confusion "What URL do I put in there?" and makes it clear they need to make the package.json HTTP-accessible too (not just the extension JS file).

Reply:
In the end, I decided against making this the URL to the package.json, and made it explicit that the user should enter the URL to the Parcel dev server. I also made this explicit in the publishing docs.

The [extension creator will be updated](https://github.com/sourcegraph/create-extension/commit/90e96c7e45b291777d7b0dfca3cabb50ec67bc7b) to make sure the package.json is HTTP-accessible when running `npm run serve`.

Since parcel can only serve from dist/, our code assumes that the package.json is served from inside of dist/ (through a symlink, as set up by the extension creator), and that the relative path to the bundle should be fixed accordingly. I'd rather hide that complexity from the user.